### ISSUE TITLE
Shortcut - Group chat counter under Shortcut appears differently than LHN and chat header

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#39052]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.57-0
**Reproducible in staging?:** y
**Reproducible in production?:** new feature
**Issue reported by:**

## Action Performed:
Precondition:
- App is in light mode.
1. Go to staging.new.expensify.com.
2. Create a group chat with at least three people.
3. Create a split bill in the group chat.
4. Go to FAB.

## Expected Result:
The group chat counter under Shortcut appears the same as the counter in LHN and chat header.

## Actual Result:
The group chat counter under Shortcut is black in light mode, while it is white in LHN and chat header.

## Workaround:
n/a

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/1fba1aef-49db-48aa-9d49-2f420c5d2da4

![image](https://github.com/Expensify/App/assets/93399543/d0d26214-b044-4842-8cb7-44a42471bd08)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
We think that this bug might be related to #vip-vsb

**Comment by DrLoopFall** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Group chat counter under Shortcut appears differently than LHN and chat header

### What is the root cause of that problem?
We are using different style for small avatar, `avatarInnerTextSmall`, which has `color: theme.text`, but for large avatar we are using `avatarInnerText` which has `color: theme.textLight`
https://github.com/Expensify/App/blob/d89fa9aa957bfb6777f41b5f2db376509589d522/src/components/MultipleAvatars.tsx#L294

### What changes do you think we should make in order to solve the problem?
Update `avatarInnerTextSmall` to use `color: theme.textLight`
```diff
        avatarInnerTextSmall: {
-           color: theme.text,
+           color: theme.textLight,
            fontSize: variables.fontSizeExtraSmall,
            lineHeight: undefined,
            marginLeft: -2,
            textAlign: 'center',
        },
```

**Comment by cristipaval** [comment]
Removing the deploy blocker label given that it is a minor UI issue that doesn't break the functionality

**Comment by cristipaval** [comment]
Tagging @Expensify/design to confirm this is a bug and to clarify the expected specs.

**Comment by dannymcclain** [comment]
So I think this _is_ a bug, but I think the actual bug is in the LHN and header, not in the Shortcut.

In dark mode, the `+X` counter uses a background color of `icons` with a text color of `text`. But in light mode, the counter uses a background color of `icons` with `#FFF` as the text color (white). We don't typically use pure white in the interface, so I'm thinking the counter in the Shortcut is actually the right version (it looks like it's using `icons` for the background color and `text` for the text color like in dark mode, but obviously they are the light mode versions of those variables).

cc'ing @shawnborton because he might know what's up!

**Comment by shawnborton** [comment]
Hmm yeah, that does seem like a bug and I agree with where you landed Danny.

**Comment by cristipaval** [comment]
Thank you, @dannymcclain and @shawnborton! I'll make this one external to get it fixed!

**Comment by dannymcclain** [comment]
Oh dang! I think you're right @paultsimura!

**Comment by shawnborton** [comment]
Yup, for new group chats moving forward we'll have that, though I think old group chats will still have the old avatars?

**Comment by cristipaval** [comment]
Ok, so it looks like we're still waiting for proposals to fix as @dannymcclain landed [here](https://github.com/Expensify/App/issues/39052#issuecomment-2023989753).

**Comment by dannymcclain** [comment]
>We want the LHN and header version to look like the one in the Shortcut?

Yeah that's what I'm thinking. So that would be like this:

![image](https://github.com/Expensify/App/assets/7516624/f636eaae-83f1-4fe1-b464-602ad3a84fbb)

**Comment by DrLoopFall** [comment]
## Proposal
[Updated](https://github.com/Expensify/App/issues/39052#issuecomment-2022922881)

cc @thesahindia

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by DrLoopFall** [comment]
@mvtglobally 

The issue is reproducible when creating split bills from existing group chats (which uses the old avatar style)

**Comment by DrLoopFall** [comment]
@thesahindia
The PR #40175 is ready for review.

**Comment by DrLoopFall** [comment]
Hi @cristipaval,
The payment for this issue is still pending, can you please process it?

**Comment by cristipaval** [comment]
@sakluger could you please help with the payments here? 🙏

**Comment by JmillsExpensify** [comment]
$250 approved for @thesahindia
